packages/frontend/e2e/src/app.po.ts

```diff
@@ -0,0 +1,11 @@
+import { browser, by, element } from 'protractor';
+
+export class AppPage {
```

Comment:
what does the `.po` extension mean?

Reply:
I also am wondering what .po means

Reply:
**.po** files contain **page objects**. Basically, they are classes that help us access specific elements in the DOM of the page that is being tested. (https://angular.io/guide/testing#use-a-page-object).
We can do that directly in the test files, but angular advises to encapsulate the entire logic in those **.po** files.